app/frontend/state.ts

```diff
@@ -61,7 +62,7 @@ export interface State {
     mnemonicInputError: {code: InternalErrorReason} | null
     formIsValid: boolean
   }
-  hwWalletName?: string
+  cryptoProviderInfo?: CryptoProviderInfo
```

Comment:
Not sure if it relates exactly to cryptoProviderInfo, but when developing with `ADALITE_DEVEL_AUTO_LOGIN = true` and trying to log out, the app does not stay logged out but immediately logs user in again, making it possible to actually log out.

Reply:
I see, good catch - that's actually a side-effect of a refactor done in https://github.com/vacuumlabs/adalite/pull/1206 where "soft" logout was replaced by full page refresh to simplify reasoning about the state which can be in a weird intermediary state as you overwrite the wallet data at once which broke assumptions around strict null checks.

I don't see a nice way to solve it right now and I would not really consider worth it solving it soon as it's a development only issue and you can still test logout properly, just with autologin off